react/src/Root/App/DistrictOverview/index.tsx

```diff
@@ -107,14 +112,150 @@ class DistrictOverview extends React.PureComponent<MyProps, State> {
         });
     }
 
+    private getLabelGeoJson = memoize((metadata?: Metadata) => {
```

Comment:
Instead of `private getLabelGeoJson = memoize((metadata?: Metadata)`, we should use this form:
`private getLabelGeoJson = memoize((metadata: Metadata | undefined)`